packages/react-search-ui/README.md

```diff
@@ -40,58 +89,160 @@ export default function App() {
     <SearchProvider config={config}>
       {() => (
         <div className="App">
-          <Layout header={<SearchBox />} bodyContent={<Results />} />
+          <Layout
+            header={<SearchBox />}
+            bodyContent={<Results titleField="title" urlField="nps_link" />}
+          />
         </div>
       )}
     </SearchProvider>
   );
 }
 ```
 
-### SearchProvider <a id="searchprovider"></a>
+<a id="searchuiconfig"></a>
+
+## Configuring Search UI
 
-The SearchProvider is the core engine behind Search UI. It provides [actions](../search-ui/README.md#actions)
-and [state](../search-ui/README.md#state) in a React [Context](https://reactjs.org/docs/context.html). If
-you are familiar with Redux, you should recognize this model.
+[back](#nav)
 
-`SearchProvider` is essentially a wrapper around our [`search-ui`](../search-ui).
+Search UI is configured via the `SearchProvider`.
+
+Example:
+
+```jsx
+const config = {
+  apiConnector: connector,
+  disjunctiveFacets: ["acres"],
+  disjunctiveFacetsAnalyticsTags: ["Ignore"],
+  search_fields: {
+    title: {},
+    description: {}
+  },
+  result_fields: {
+    title: {
+      snippet: {
+        size: 100,
+        fallback: true
+      }
+    },
+    nps_link: {
+      raw: {}
+    },
+    description: {
+      snippet: {
+        size: 100,
+        fallback: true
+      }
+    }
+  },
+  facets: {
+    states: { type: "value", size: 30 },
+    acres: {
+      type: "range",
+      ranges: [
+        { from: -1, name: "Any" },
+        { from: 0, to: 1000, name: "Small" },
+        { from: 1001, to: 100000, name: "Medium" },
+        { from: 100001, name: "Large" }
+      ]
+    }
+  }
+};
+
+return (
+  <SearchProvider config={config}>
+    {() => (
+      <div className="App">
+        <Layout
+          header={<SearchBox />}
+          bodyContent={<Results titleField="title" urlField="nps_link" />}
+        />
+      </div>
+    )}
+  </SearchProvider>
+);
+```
+
+<a id="searchprovider"></a>
+
+### SearchProvider
+
+`SearchProvider` is what connects Search UI to your components. Components you place
```

Comment:
> As you'll see in the [Components](#components) section, Search UI provides an extensive library of
Components which you can use to build a search experience.
However, before you can start dropping in those Components, you'll first need to create a
`SearchProvider`, which will tie all of your Components together so that they
work as a cohesive application.
>
> `SearchProvider` ultimately converts user actions into API calls to your Search API, so you'll need to
configure it so that it knows where to direct those API calls. A comprehensive list of
configuration options are included below in the [Search UI Configuration](#config) section.

Reply:
Most excellent. 👌